core/request.go

```diff
@@ -169,46 +179,59 @@ func (r *RequestAccessor) RequestFromEvent(ctx context.Context, req events.APIGa
 		log.Println(err)
 		return nil, err
 	}
-
 	for h := range req.Headers {
 		httpRequest.Header.Add(h, req.Headers[h])
 	}
+	return httpRequest, nil
+}
 
-	apiGwContext, err := json.Marshal(req.RequestContext)
+// AddToHeader adds API gateway context and stage variables to headers on http request
+func AddToHeader(req *http.Request, apiGwRequest events.APIGatewayProxyRequest) (*http.Request, error) {
```

Comment:
I don't think this should be exported. Putting the context in headers is not-ideal.

Reply:
Agreed. I had it exported because I thought I would need in Proxy, but ended up going about it differently. Fixed.